doc/ref_model/chapters/chapter08.rst

```diff
@@ -202,13 +202,30 @@ The API and Cloud Brokerage models are illustrated in the following diagrams:
 Requirements, Reference Architecture & Industry Standards Intersect
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The Communcations Service Provider is both a provider and consumer of Cloud based services.
-When the CSP is actings as:
+The Communcations Service Provider (CSP) is both a provider and consumer of Cloud based services. When the CSP is acting as:
+- consumer of cloud services, the typical consideration is total cost of ownership as the consumption is usually to support internal business operations: BSS/OSS systems;
```

Comment:
Doesn't show as bullets any more.

Reply:
@ulikleber experience/testing has shown that you need to have a blank line before the start of the bulleted list. I will update. Thank you for feedback.